Makefile

```diff
@@ -5,6 +5,11 @@ DOCKER_IMAGE_REPO ?= quay.io/thanos/thanos
 DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))-$(shell date +%Y-%m-%d)-$(shell git rev-parse --short HEAD)
 DOCKER_CI_TAG     ?= test
 
+DOCKER_ARCHS       ?= amd64 arm64
+BUILD_DOCKER_ARCHS = $(addprefix docker-build-,$(DOCKER_ARCHS))
```

Comment:
This is some Makefile magic I never seen before, do you mind explaining what it does? ;p preferably in comment?

Reply:
I will help fix :)

Add them.